tasks/rubies.yml

```diff
@@ -1,17 +1,11 @@
 ---
 
-- name: Detect if rubies are installed
-  command: '{{ rvm1_rvm }} {{ item }} do true'
-  changed_when: False
-  failed_when: False
-  register: detect_rubies
-  with_items: '{{ rvm1_rubies }}'
-  when: rvm1_rubies is defined and rvm1_rubies | length > 0
-
 - name: Install rubies
-  command: '{{ rvm1_rvm }} install {{ item.item }} {{ rvm1_ruby_install_flags }}'
-  when: rvm1_rubies is defined and rvm1_rubies | length > 0 and item.rc|default(0) != 0
-  with_items: '{{ detect_rubies.results }}'
+  command: '{{ rvm1_rvm }} install {{ item }} {{ rvm1_ruby_install_flags }}'
```

Comment:
This looks like the code in #219!

Please check-out my comment at https://github.com/rvm/rvm1-ansible/pull/219#discussion_r464399241 in that case!

Reply:
This is indeed, see commit history and main description of this PR.

This PR is pending on #219. I mark it as Draft to make things clearer.